From 268d948744fa4d287737f109cd48c5c640e41d40 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martinpitt@gnome.org>
Date: Mon, 20 Aug 2012 06:02:31 +0200
Subject: [PATCH] Bump max time in test_method_calls_on_proxy() test

On slower platforms, the overhead of the 240 D-BUS Sleep calls is larger than
the current maximum of 6 seconds. A run on a Panda board sometimes fails with

  ERROR:/build/buildd/glib2.0-2.33.8/./gio/tests/gdbus-threading.c:409:test_method_calls_on_proxy:
  assertion failed (elapsed_msec < 6000): (7365 < 6000)

Bump maximum time to 8 seconds to be more resilient to this.

https://bugzilla.gnome.org/show_bug.cgi?id=682222
---
 gio/tests/gdbus-threading.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/gdbus-threading.c b/gio/tests/gdbus-threading.c
index c1fc97e..728b70e 100644
--- a/gio/tests/gdbus-threading.c
+++ b/gio/tests/gdbus-threading.c
@@ -406,7 +406,7 @@ test_method_calls_on_proxy (GDBusProxy *proxy)
 
       /* elapsed_msec should be 4000 msec +/- change for overhead/inaccuracy */
       g_assert_cmpint (elapsed_msec, >=, 3950);
-      g_assert_cmpint (elapsed_msec,  <, 6000);
+      g_assert_cmpint (elapsed_msec,  <, 8000);
 
       g_print (" ");
     }
-- 
1.7.10.4

